projects/ontimize-web-ngx/src/lib/services/ontimize/ontimize-export.service.ts

```diff
@@ -45,15 +46,17 @@ export class OntimizeExportService extends OntimizeBaseService implements IExpor
     return headers;
   }
 
-  public exportData(data: any, format: string, entity?: string): Observable<any> {
+  public exportData(format: string): Observable<any> {
```

Comment:
It is a **breaking change** isn't it? Maybe someone overrides this service method and it will not work anymore.